dynamics-nav/SELECTLATESTVERSION-Function--Database-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SELECTLATESTVERSION Function (Database)"
+description: SELECTLATESTVERSION Function (Database) Forces the latest version of the database to be used. This topic contains information on the SELECTLATESTVERSION Function (Database) syntax, remarks, and additional information.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.